omod/src/main/webapp/pages/htmlform/NullPointerException.gsp

```diff
@@ -0,0 +1,12 @@
+<html>
+   <body>
+   <p>Application has encountered an error. Please contact support on ...</p>
+     <% out << " ooppsss!!!! Active drugs cannot be edittted!" %>
```

Comment:
what is this  for ?? and how do you expect it to work anyway??

Reply:
This should ideally be returned as an error page like returned in the controller class

Reply:
Yes, instead of "active drugs cannot be editted", don't we want to just return the message below?

Reply:
Sure thanks have collected the error return message